templates/gatsby-wordpress/files/frontend/.platform.app.yaml

```diff
@@ -0,0 +1,38 @@
+# .platform.app.yaml
+
+# The name of this application, which must be unique within a project.
+name: 'gatsby'
+
+# The type key specifies the language and version for your application.
+type: 'nodejs:12'
+
+# The hooks that will be triggered when the package is deployed.
+hooks:
+    # Build hooks can modify the application files on disk but not access any services like databases.
+    post_deploy: |
+      npm run build
+
+# The size of the persistent disk of the application (in MB).
+disk: 1280
+
+# relationships:
+#   backend: "wordpress:http"
```

Comment:
This isn't necessary?  I find that surprising.  If so, it should just be omitted entirely.